src/main/java/com/direwolf20/buildinggadgets/common/util/blocks/RegionSnapshot.java

```diff
@@ -98,7 +106,7 @@ private static NBTTagCompound serialize(NBTTagCompound tag, RegionSnapshot snaps
             frames.add((((streak - 1) & 0xff) << 24) | (mapPalettes.getInt(lastStreak.orElse(null)) & 0xffffff));
 
             // Regardless of which situation, we reset the counter
-            streak = 0;
+            streak = 1;
```

Comment:
You sure that this is intended? I thought we got the streaking right...
Might want to add a write/read test to verify...

Reply:
Tested with 0, it doesn't work. We would already encounter a new type of block when resetting counter.

Reply:
Ok, good to know

Reply:
> Tested with 0, it doesn't work. We would already encounter a new type of block when resetting counter.
> 
> Also streaking more than 255, or more than one streak frames doesn't work yet (in terms of [d5fc568](https://github.com/Direwolf20-MC/BuildingGadgets/commit/d5fc5683c2e8f213ebccd9f968bb7c071d74a84a))

Fixed in 25ce27d!